src/server.lisp

```diff
@@ -158,104 +158,104 @@ DISPATCH-TABLE and LOGGING-STREAM are both required arguments.  TIMEOUT is of ty
   (pzmq:with-socket receiver :dealer
     (pzmq:connect receiver pool-address)
     (loop
-      (handler-case
+       (tagbody
+          :start
           (let ((warnings (make-array 0 :adjustable t :fill-pointer 0))
-                request result reply start-time)
+                request result reply start-time identity empty-frame raw-request)
```

Comment:
The temptation to fuse this `TAGBODY` + `LET` into a `PROG` was overwhelming, but in the end I resisted.

I moved these bindings here to remove one level of nesting with `M-V-B` and to allow all the pre-request forms to wrapped in a single `HANDLER-BIND`.